Src/Support/GoogleApis/Apis/Requests/HttpRequestMessageExtenstions.cs

```diff
@@ -19,7 +19,11 @@
 using System.Net.Http.Headers;
 using System.Text;
 
+#if (NET45 || NETSTANDARD)

```

Comment:
We're only using GZipStream in a simple way, and it does indeed appear to be identical.
We'll make sure this is included in testing.